resources/jscomposition/cases/casesDetail/api/index.js

```diff
@@ -1,19 +1,25 @@
 import { api } from "../variables";
 
-export const getData = async () => {
+const getData = async () => {
   const objectsList = [];
 
   for (let i = 0; i <= 31; i += 1) {
```

Comment:
Fabio, currently you can use the API: `tasks-by-case?case_number=80&status=CLOSED` instead of fakeData